helm/templates/dashboard.yaml

```
@@ -0,0 +1,11 @@
+{{- if .Values.grafanaDashboard -}}
+apiVersion: integreatly.org/v1alpha1
+kind: GrafanaDashboard
+metadata: 
+  name: immudb-grafana-dashboard
+  labels: 
+    app: immudb
+spec:
+  url: https://raw.githubusercontent.com/codenotary/immudb/master/tools/monitoring/grafana-dashboard.json
```

We could use URL for specific immudb version here, e.g. `https://raw.githubusercontent.com/codenotary/immudb/v1.3.2/tools/monitoring/grafana-dashboard.json`

that could default to the same version as the immudb one that is used in the statefulset with an option to override, e.g.:

```suggestion
  url: https://raw.githubusercontent.com/codenotary/immudb/{{- .Values.grafanaDashboard.version | default print "v" .Chart.AppVersion -}}/tools/monitoring/grafana-dashboard.json
```